azure-pipelines.yml

```diff
@@ -64,4 +65,5 @@ jobs:
         git commit --allow-empty -m "Documentation for $(GitVersion.PreReleaseTag)"
         git push origin gh-pages
       displayName: 'Publish documentation to GitHub Pages'
-      condition: and(succeeded(), or(variables['Endjin.ForceDocPublish'], eq(variables['GitVersion.PreReleaseTag'], '')))
\ No newline at end of file
+      condition: and(succeeded(), or(variables['Endjin.ForceDocPublish'], eq(variables['GitVersion.PreReleaseTag'], '')))
+    compileTasksServiceConnection: endjin-acr-reader
```

Comment:
We need to look at some of the customisations in this YAML file:

1. Check whether they are still needed?
2. Check whether the extensibility points it uses are available in the scripted build version of the template